### PR TITLE
fix: address some minor bugs in swapHistory & wallet page loading

### PR DESCRIPTION
**Pull request by samobasquiat**
# Summary

Some minor bugs should be resolved.

Fixes # (issue)

# How did you test this change?

- [x] Wallets page loading should have a three-column layout
- [x] The amounts in the swap history page are too precise (should be rounded).
- [x] when custom slippage is empty, slippage property in best route gets NaN.

# Checklist:

- [x] I have performed a self-review of my code
- [x] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

File: widget/embedded/src/pages/WalletsPage.tsx [hunk 1]
```diff
@@ -11,11 +11,10 @@ import { useAppStore } from '../store/AppStore';
...
-  justifyContent: 'space-evenly',
-  alignItems: 'center',
+  display: 'grid',
   gap: '$10',
-  flexWrap: 'wrap',
+  gridTemplateColumns: ' repeat(auto-fill, minmax(100px, 1fr))',
+  alignContent: 'baseline',
   paddingTop: '$5',
   height: '100%',
 });
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/pages/WalletsPage.tsx:20]: To resolve the issue and maintain consistent sizes for the wallet button and its skeleton, you can retain the Flexbox styles and simply reduce the width of the wallet loading button within the UI package.

File: widget/embedded/src/pages/WalletsPage.tsx [hunk 2]
```diff
@@ -11,11 +11,10 @@ import { useAppStore } from '../store/AppStore';
...
 
 const ListContainer = styled('div', {
-  display: 'flex',
-  justifyContent: 'space-evenly',
-  alignItems: 'center',
+  display: 'grid',
   gap: '$10',
-  flexWrap: 'wrap',
+  gridTemplateColumns: ' repeat(auto-fill, minmax(100px, 1fr))',
+  alignContent: 'baseline',
```
Comment (Ikari-Shinji-re) [inline widget/embedded/src/pages/WalletsPage.tsx:17]: To resolve the issue and maintain consistent sizes for the wallet button and its skeleton, you can retain the Flexbox styles and simply reduce the width of the wallet loading button within the UI package.

File: widget/ui/src/components/Popover/Popover.tsx [hunk 3]
```diff
@@ -16,7 +16,10 @@ export const PopoverContentComponent = (
   const { container, children, ...rest } = props;
   return (
     <RadixPopover.Portal container={container}>
-      <PopoverContainer {...rest} ref={forwardedRef}>
+      <PopoverContainer
+        onOpenAutoFocus={(e) => e.preventDefault()}
+        {...rest}
```
Comment (Ikari-Shinji-re) [inline widget/ui/src/components/Popover/Popover.tsx:21]: This solution doesn't completely address the issue.

Comment (samobasquiat) [reply widget/ui/src/components/Popover/Popover.tsx:21]: It was fixed in another PR by @Ikari-Shinji-re .

File: widget/embedded/src/pages/WalletsPage.tsx (hunk 2 above)
Comment (samobasquiat) [reply widget/embedded/src/pages/WalletsPage.tsx]: changed.

**Review by Ikari-Shinji-re — APPROVED** [review]

**Review by mikasackermn — APPROVED** [review]

**Pull request merged**